x-pack/plugins/maps/public/components/layer_panel/index.js

```diff
@@ -12,8 +12,10 @@ import {
 } from '../../actions/store_actions';
 
 function mapStateToProps(state = {}) {
+  const selectedLayer = getSelectedLayer(state);
   return {
-    selectedLayer: getSelectedLayer(state)
+    selectedLayer,
+    styleProperties: selectedLayer.getCurrentStyle().getProperties()
```

Comment:
we probably don't need all that. cf. `hasDirtyState` selector should be enough (probably)